frontend/packages/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  root: true,
+  extends: [
+    'plugin:console/react',
+    'plugin:console/typescript',
+    // TODO enable when we stop using jest with jasmine types
```

Comment:
AFAIK, Jest has custom-built Jasmine version included in it. Did that change recently?

What do you mean by "stop using jest with jasmine types"?

Reply:
Jest supports jasmine API but have equivalent jest API. A bunch of rules suggest prefer doing one way over another. Or we just evaluate turning certain rules off.

Reply:
Got it, I'd prefer us moving to Jest APIs given the opportunity.

We can also consider using [Jest codemods](https://jestjs.io/docs/en/migration-guide#jest-codemods) if that makes sense.